doc/ca65.sgml

```diff
@@ -2699,6 +2699,20 @@ Here's a list of all control commands and a description, what they do:
     at character is not allowed to start an identifier, even with this
     feature enabled.
 
+  <tag><tt>bracket_as_indirect</tt><label id="bracket_as_indirect"></tag>
+
+    Use <tt>[]</tt> intead of <tt>()</tt> for the indirect addressing mode.
+    Example:
+
+    <tscreen><verb>
+        lda     [$82]
+        lda     [$82,x]
+        lda     [$82],y
```

Comment:
Is this supposed to be applicable to JMP (...) too? If yes:
- Did you tested it? 
- It should be added here.